### DISCUSSION TITLE
Getting 404 error even though my repo is named correctly, is public, and has the settings

### DISCUSSION THREAD

**Discussion by anish-palakurthi** [community/community#59678] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I followed an online tutorial but am still getting a 404 not found error. Does anyone know what the issue might be?

**Comment by GwenaelAlo** [comment]
Hi, I have the same problem. Did you find the solution?
the repository name is ok, I put index.html.... and I got the confirmation on the W3C.

(i'm sorry my english is not very good)
